### DISCUSSION TITLE
pac connection update issue

### DISCUSSION THREAD

**Discussion by lar-mar** [microsoft/powerplatform-build-tools#552] [General]
I was already looking for months for a way to update the client secret of a ServicePrincipal used in a Dataverse connection.
I found a workaround but now I'm glad there is an official way to do it via CLI.

I tried to update a connection using "pac connection update" and also passed the necessary params.
Unfortunately I only get the error message: The remote name could not be resolved: 'default572cf4e49b264a0780770db7366890.25.environment.api.powerplatform.com'

Any ideas how to resolve it?

**Comment by petrochuk** [comment]
This looks like valid Power Platform API url for your environment. I would suggest opening support ticket and attaching log file to it

**Comment by lar-mar** [comment]
@petrochuk 
Ok, I'll do it.
